README.md

```diff
@@ -49,6 +49,7 @@ Other clients:
 - [logstash-input-pulsar](https://github.com/se7enkings/logstash-input-pulsar): A logstash input that receives data from Pulsar.
 - [pulsar-beat-output](https://github.com/streamnative/pulsar-beat-output): [Elastic Beats](https://github.com/elastic/beats) to Apache Pulsar.
 - [pulsar-flume-ng-sink](https://github.com/streamnative/pulsar-flume-ng-sink): An [Apache Flume](https://github.com/apache/flume) Sink implementation to publish data to Pulsar.
+- [pinterest/singer](https://github.com/pinterest/singer): A high-performance, reliable and extensible logging agent for uploading data to Kafka, Pulsar, etc.
```

Comment:
```suggestion
- [pinterest/singer](https://github.com/pinterest/singer): A high-performance, reliable and extensible logging agent for uploading data to Kafka and Pulsar.
```

Reply:
Or may be you can use "Kafka or Pulsar".

Reply:
@Jennifer88huang I just copied the description from the project itself - https://github.com/pinterest/singer . I am not sure we should change it to "and" or "or". Neither sounds correct to me. I think the description was meant to say it can upload the log data to many other similar systems like Pulsar and Kafka.